client/my-sites/media-library/list-plan-upgrade-nudge.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import UpgradeNudgeExpanded from 'blocks/upgrade-nudge-expanded';
+import { PLAN_PREMIUM, FEATURE_VIDEO_UPLOADS, FEATURE_AUDIO_UPLOADS } from 'lib/plans/constants';
+
+export const MediaLibraryUpgradeNudge = ( { translate, filter } ) => {
+	let title = translate( 'Upgrade to a Premium Plan and Enable VideoPress' );
+	let subtitle = translate( `By upgrading to a Premium Plan you'll enable VideoPress support on your site.` );
```

Comment:
Is your eslint setup in your editor? We should use single strings here. 

We can also run `make eslint-branch` locally to lint the related files